__tests__/app.test.js

```diff
@@ -16,4 +16,20 @@ describe ('/api/does-not-exist', () => {
             expect(body.msg).toBe('Not found')
         })
     })
+})
+
+describe ('/api/topics', () => {
+    test('GET 200: responds with an array of topic objects, each with slug and description properties', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+            const { topics } = body
+            expect(topics.length).toBe(3)
+            topics.forEach((topic) => {
+                expect(typeof topic.description).toBe('string')
+                expect(typeof topic.slug).toBe('string')
+            })
```

Comment:
Nice first test! 
It could also be worth thinking about any errors you may encounter here (nothing you need to add now but have a think about it 🙂)